src/test/kotlin/racing/CarRacingTest.kt

```diff
@@ -14,75 +19,77 @@ class CarRacingTest {
     fun `차 리스트 생성하고 차 리스트에 각자 이름이 제대로 들어간다`() {
         val carRacing = CarRacing()
         val cars: List<Car> = carRacing.createCars(listOf("현대", "samsung", "L9"))
-        assertThat(cars[0].name).isEqualTo("현대")
-        assertThat(cars[1].name).isEqualTo("samsung")
-        assertThat(cars[2].name).isEqualTo("L9")
+
+        assertThat(cars).containsExactlyInAnyOrder(Car("현대"), Car("samsung"), Car("L9"))
     }
 
     @Test
     fun `우승자 테스트 a만 우승`() {
         val carRacing = CarRacing()
-        val moveCount = 5
-        val carA = Car("a").apply { repeat(moveCount) { turnOfPosition.add(5) } }
-        val carB = Car("b").apply { repeat(moveCount) { turnOfPosition.add(1) } }
-        val carC = Car("c").apply { repeat(moveCount) { turnOfPosition.add(1) } }
-        val carD = Car("d").apply { repeat(moveCount) { turnOfPosition.add(2) } }
-        val carE = Car("e").apply { repeat(moveCount) { turnOfPosition.add(2) } }
+        val carA = Car("a").also { it.go { 2 } }
```

Comment:
```suggestion
        val carA = Car("a", position = 2)
```
https://github.com/next-step/kotlin-racingcar/pull/526#discussion_r872564317 에 대한 추가 피드백입니다. 위 코드로 개선해 볼까요?

Reply:
생성자에서 position = 2를 두는 것은 오롯이 테스트를 위함인가요?

아니면 default 위치를 유연하게 하기 위함인가요?

Reply:
테스트를 위해서 입니다.